lizard_auth_server/views_api.py

```diff
@@ -300,3 +300,121 @@ def get_organisations(self, portal):
             'organisations': [
                 organisation.as_dict()
                 for organisation in models.Organisation.objects.all()]}
+
+
+class GetRolesView(FormView):
```

Comment:
Zelf hou ik niet zo van dat 'get' gebeuren in namen. Maar goed, er zijn al twee GetUsersView classes enzo. Maar de andere views heten niet zo. Wat vind je ervan deze gewoon "RolesView" te noemen?

"get" wordt in python voornamelijk gebruikt voor speciale methodes zoals "get_field()" op models om programmatisch aan een veldnaam te komen enzo.